Mapping Tools/Classes/SystemTools/Settings.cs

```diff
@@ -94,6 +92,38 @@ public class Settings : INotifyPropertyChanged {
             }
         }
 
+        private bool _smartQuickRunEnabled;
+        public bool SmartQuickRunEnabled {
+            get => _smartQuickRunEnabled;
+            set {
+                if (_smartQuickRunEnabled == value) return;
+                _smartQuickRunEnabled = value;
+                OnPropertyChanged();
+            }
+        }
+
+        public static IEnumerable<SingleQuickRunEnum> SingleQuickRunEnumerable => Enum.GetValues(typeof(SingleQuickRunEnum)).Cast<SingleQuickRunEnum>();
+        private SingleQuickRunEnum _singleQuickRunTool;
+        public SingleQuickRunEnum SingleQuickRunTool {
+            get => _singleQuickRunTool;
+            set {
+                if (_singleQuickRunTool == value) return;
+                _singleQuickRunTool = value;
+                OnPropertyChanged();
+            }
+        }
+
+        public static IEnumerable<MultipleQuickRunEnum> MultipleQuickRunEnumerable => Enum.GetValues(typeof(MultipleQuickRunEnum)).Cast<MultipleQuickRunEnum>();
```

Comment:
what are these static enum lists doing in settings? Settings is reserved for actual non-static stuff that must get saved.